spec/AutomatedLinkAcceptance+services.yaml

```diff
@@ -93,170 +105,191 @@ clients:
-            uuid: xx-1-0-0-op-c-bm-alt-1-0-0-000
+            uuid: ALA-1-0-0-op-c-bm-alt-2-0-0-000
           - service-name: /v1/update-ltp
-            uuid: xx-1-0-0-op-c-bm-alt-1-0-0-001
+            uuid: ALA-1-0-0-op-c-bm-alt-2-0-0-001
           - service-name: /v1/delete-ltp-and-dependents
-            uuid: xx-1-0-0-op-c-bm-alt-1-0-0-002
+            uuid: ALA-1-0-0-op-c-bm-alt-2-0-0-002
           - service-name: /v1/update-fc
-            uuid: xx-1-0-0-op-c-bm-alt-1-0-0-003
+            uuid: ALA-1-0-0-op-c-bm-alt-2-0-0-003
           - service-name: /v1/update-fc-port
-            uuid: xx-1-0-0-op-c-bm-alt-1-0-0-004
+            uuid: ALA-1-0-0-op-c-bm-alt-2-0-0-004
           - service-name: /v1/delete-fc-port
-            uuid: xx-1-0-0-op-c-bm-alt-1-0-0-005
+            uuid: ALA-1-0-0-op-c-bm-alt-2-0-0-005
         individual:
       service:
         basic:
-          ## TODO: Add Clients specific to this application, but addressing all other applications
+
         individual:
-          ## TODO: Add Clients specific to this application
+
 
   - http-client:
       application-name: OperationKeyManagement
-      release-number: 1.0.0
-      uuid: xx-1-0-0-http-c-is-okm-1-0-0-000
+      release-number: 2.0.0
+      uuid: ALA-1-0-0-http-c-okm-2-0-0-000
     tcp-client:
-      ip-address: 10.118.125.157
-      tcp-port: 1006
-      uuid: xx-1-0-0-tcp-c-is-okm-1-0-0-000
+      remote-protocol: http
+      ip-address: 1.1.3.14
+      tcp-port: 3014
+      uuid: ALA-1-0-0-tcp-c-okm-2-0-0-000
     operation-clients:
       own-oam:
         basic:
         individual:
       service:
         basic:
-          ## TODO: Add Clients specific to this application, but addressing all other applications
+
         individual:
-          ## TODO: Add Clients specific to this application
+
 
   - http-client:
-      application-name: ## TODO: Add Name of application to be addressed
-      release-number: ## TODO: Add Release number of application to be addressed
-      uuid: ## TODO: Add UUID of the HttpClient
+      application-name: MicroWaveDeviceInventory
+      release-number: 1.0.0
+      uuid: mwdi-1-0-0-http-s-is-000
     tcp-client:
-      ip-address: ## TODO: Add IP address of application to be addressed
-      tcp-port: ## TODO: Add TCP port of application to be addressed
-      uuid: ## TODO: Add UUID of the TcpClient
+      remote-protocol: http
+      ip-address: 1.1.4.4
+      tcp-port: 4004
+      uuid: mwdi-1-0-0-tcp-s-is-000
     operation-clients:
       own-oam:
         basic:
         individual:
       service:
         basic:
-          ## TODO: Add Clients specific to this application, but addressing all other applications
+
         individual:
-          ## TODO: Add Clients specific to this application
+          - service-name: /node={mount-name}/yang-ext:mount/core-model-1-4:control-construct?content=config&fields=logical-termination-point(uuid;name;layer-protocol(local-id;layer-protocol-name))
```

Comment:
in your service-name you have provided the service name from the mwdi together with a filter (_?content=..._).  

As I have not come across this before, I am not sure if you are allowed to add the filter here as part of the service-name (or operation-name, once this has been corrected) or if this needs to be specified in another way or elsewhere. (But it looks reasonable to me.)  

Maybe we can check this with Thorsten to be on the safe side.

Reply:
update 13.02.2023: we discussed that we will support the fields filter, but not the content filter. So the content=nonconfig part can be removed from the query string.

Reply:
* Maintained the fields filter and removed the  ?content=nonconfig and ?content=config filters
* Corrected the format of the operation-name uuids